ruby/README.md

```diff
@@ -0,0 +1,77 @@
+# Ruby
+
+- Avoid conditional modifiers (lines that end with conditionals).
+- Avoid multiple assignments per line (`one, two = 1, 2`).
+- Avoid organizational comments (`# Validations`).
+- Avoid ternary operators (`boolean ? true : false`). Use multi-line `if`
+  instead to emphasize code branches.
+- Avoid bang (!) method names. Prefer descriptive names. [#122]
```

Comment:
Came back to the guide to read this bullet and noticed that in the Thoughtbot repo this links to a GH issue where peeps are talking about it. Do we want to link to the full issue `https://github.com/thoughtbot/guides/pull/122` or remove the link reference?

Reply:
Good catch! I think I'll remove it. Unless someone else feels strongly otherwise

Reply:
To me it doesn't have a lot of value until the end of all the comments. I think if we want later we could later add our own context to reinforce the reason.